src/custom/constants/index.ts

```diff
@@ -27,13 +29,27 @@ export const BARN_URL = `barn.${PRODUCTION_URL}`
 
 const DISABLED_WALLETS = /^(?:WALLET_LINK|COINBASE_LINK|FORTMATIC|Portis)$/i
 
+const ADDED_WALLETS: { [key: string]: WalletInfo } = {
+  TALLY: {
+    connector: injected,
+    name: 'Tally',
+    iconURL: TALLY_ICON_URL,
+    description: 'Connect with Tally Wallet',
+    href: null,
+    color: '#D59B4B',
+  },
+}
+
 // Re-export only the supported wallets
-export const SUPPORTED_WALLETS = Object.keys(SUPPORTED_WALLETS_UNISWAP).reduce((acc, key) => {
-  if (!DISABLED_WALLETS.test(key)) {
-    acc[key] = SUPPORTED_WALLETS_UNISWAP[key]
-  }
-  return acc
-}, {} as { [key: string]: WalletInfo })
+export const SUPPORTED_WALLETS = {
```

Comment:
The wallet order is determined here.
Because I could not edit src I had to come up with some workaround.

Another option would be to make a copy in custom of constants/wallet.ts so ordering can be done manually within this SUPPORTED_WALLETS variable. (and just not use the one from src)

Reply:
Indeed, not to worry.
I realized after reading the code that the order is not intentional.
If we want we can change that at a later time.